app/scripts/auto-type/auto-type-emitter-factory.js

```diff
@@ -1,10 +1,13 @@
 import { Launcher } from 'comp/launcher';
 
 const AutoTypeEmitterFactory = {
-    create(callback) {
+    create(callback, windowID) {
         if (Launcher && Launcher.autoTypeSupported) {
             const { AutoTypeEmitter } = require('./emitter/auto-type-emitter-' +
                 Launcher.platform());
+            if (Launcher.platform() === 'linux') {
```

Comment:
This check is not needed: if `windowId` is provided, let's use it. Other systems may use it too in future.